fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/physical/PhysicalSink.java

```diff
@@ -54,4 +54,8 @@ public List<Slot> computeOutput() {
                 .map(NamedExpression::toSlot)
                 .collect(ImmutableList.toImmutableList());
     }
+
+    public List<NamedExpression> getOutputExprs() {
+        return outputExprs;
+    }
```

Comment:
remove override function in `org.apache.doris.nereids.trees.plans.physical.PhysicalResultSink`